examples/analytics-vendors.amp.html

```diff
@@ -95,6 +96,7 @@
       <option>top100</option>
       <option>topmailru</option>
       <option>umenganalytics</option>
+      <option>upscore</option>
```

Comment:
This should be alphabetical. Could you move `upscore` and `umenganalytics` below `treasuredata` ? Thank you!